### PR TITLE
uwsgi: fix wrong target names for plugins

### PR DESCRIPTION
**Pull request by pfpulux**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by pfpulux** [comment]
What should be use as license tag for [this](https://github.com/unbit/uwsgi/blob/master/LICENSE)?

**Comment by paper42** [comment]
@pfpulux check #31463, what exactly does this PR fix?

**Comment by pfpulux** [comment]
At the moment the subpackages `uwsgi-cgi`and `uwsgi-python3` write not in the folder `/usr/lib/uwsgi`, he write the file `/usr/lib/uwsgi`, so the plugins can't be found. This PR fix this.

Now the plugins for the subpackages are in place and can be found:

```
/usr/lib/uwsgi
├── cgi_plugin.so
└── python3_plugin.so
```

**Comment by paper42** [comment]
Oh, sorry that I missed that, the version I have installed on my PC has the correct paths, I must have changed something and not checked. Could you remove the $_libdir variable now? Also maybe mentioning that the xlint license warning is a false warning would be nice.

File: srcpkgs/uwsgi/template [hunk 1]
```diff
@@ -44,13 +44,15 @@ do_install() {
...
 		vcopy python3_plugin.so $_libdir
 	}
 }
 
 uwsgi-cgi_package() {
 	short_desc="$short_desc (CGI plugin)"
 	pkg_install() {
+		vmkdir $_libdir
 		vcopy cgi_plugin.so $_libdir
 	}
```
Comment (ericonr) [inline srcpkgs/uwsgi/template:57]: What do you think about using `vinstall`? I think it's more correct.

Comment (ericonr) [reply srcpkgs/uwsgi/template:57]: Instead of `vmkdir` + `vcopy`, just `vinstall`.

**Comment by sgn** [comment]
> What should be use as license tag for [this](https://github.com/unbit/uwsgi/blob/master/LICENSE)?

`GPL-2.0-only WITH GCC-exception-2.0`

**Pull request merged**